explorer/src/pages/AccountDetailsPage.tsx

```diff
@@ -265,7 +276,15 @@ function DetailsSections({
         </div>
       )}
       {<InfoSection account={account} />}
-      {<MoreSection account={account} tab={moreTab} tabs={tabs} />}
+      <React.Suspense
+        fallback={<LoadingCard message="Loading account sections" />}
+      >
```

Comment:
I see you've wrapped the _entire_ more section in Suspense. This section will suspend while the Anchor tabs are deciding if they are a thing or not.

Another choice would be to suspend _just_ the tab bar, and let everything else render.

Another choice would be to suspend _just_ each Anchor tab component – to let them pop in if they decide they're a thing, but otherwise not block the rest of the UI.

I'll let you decide. Imagine that getting an answer on whether a `programId` is an Anchor program takes a full 3 seconds, and choose where you think the suspense boundary should go.

Reply:
Okay this makes sense -- for now will just pop AnchorLink if it becomes available. Future PR can add some UI indicator (special spinning loader for tab bar?)